numpy/random/_pcg64.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Dict, Literal, Union
+
+from numpy.random.bit_generator import BitGenerator as BitGenerator
+from numpy.random.bit_generator import SeedSequence as SeedSequence
```

Comment:
```suggestion
from numpy.random.bit_generator import BitGenerator
from numpy.random.bit_generator import SeedSequence
```
You only want `from x import y as y` if `y` should be importable from both `x` and the module containing the import statement.
In other words: you're currently saying that it is completely fine to import `BitGenerator` from the `_pcg64` module.

Reply:
Thanks. Didn't know that.